quiche/src/lib.rs

```diff
@@ -2955,6 +2955,10 @@ impl Connection {
 
         self.ack_eliciting_sent = false;
 
+        // Reset pacer and start a new burst when a valid
+        // packet is received.
+        self.paths.get_mut(recv_pid)?.recovery.pacer.reset(now);
+
```

Comment:
Why do you want to reset pacer here ?

Reply:
https://github.com/cloudflare/quiche/pull/1473#discussion_r1160125794

Reply:
See my comment in above PR.